cluster/update-storage-objects.sh

```diff
@@ -74,7 +74,7 @@ do
     # TODO hopefully we can remove this once we use dynamic discovery of gettable/updateable
     # resources.
     set +e
-    instances=( $("${KUBECTL}" get "${resource}" --namespace="${namespace}" -o go-template="{{range.items}}{{.metadata.name}} {{end}}"))
+    IFS=" " read -r -a instances <<< "$("${KUBECTL}" get "${resource}" --namespace="${namespace}" -o go-template="{{range.items}}{{.metadata.name}} {{end}}")"
```

Comment:
Which shellcheck failure are you addressing here?

Reply:
```
In ./cluster/update-storage-objects.sh line 78:
    instances=( $("${KUBECTL}" get "${resource}" --namespace="${namespace}" -o go-template="{{range.items}}{{.metadata.name}} {{end}}"))
                ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
```